packages/cozy-konnector-libs/src/libs/normalizeFilename.js

```diff
@@ -0,0 +1,42 @@
+/**
+ * Returns the given name, replacing characters that could be an issue when
+ * used in a filename with underscores.
+ *
+ * Replaced characters include:
+ *
+ * - Those forbidden on one or many popular OS or filesystem: `<>:"/\|?*`
+ * - Those forbidden by the cozy-stack `\0`, `\r` and `\n`
+ * - Spaces and tabs (while those should work, preventing them may magically
+ *   fix issues with other softwares)
+ *
+ * A warning will be logged in case the result doesn't make sense.
```

Comment:
Oops, should be `An exception will be thrown in case there is not any filename-compatible character in the given string`.

I wonder whether we could enforce some kind of default name, but sounds convoluted.

Reply:
I assume it's the responsibility of the developer to provide enough input to generate a valid filename... Makes sense?